core/standalone/README.md

```diff
@@ -380,6 +386,25 @@ This shows an output like below indicating that KubernetesContainerFactory based
 }
 ```
 
+#### Launching OpenWhisk standalone from Docker:
+
+You can launch the standalone openwhisk from the docker image with:
+
+```bash
+docker run --rm -d \
```

Comment:
May be we add a simple shell script which wraps this command and people can then simply use `bash <(curl -s https://s.apache.org/openwhisk.sh)` where we can use use a short url for this script via [Apache Url Shortner](https://s.apache.org/)!!

Reply:
May be drop making it daemon by default as its useful to see logs at launch itself for new users who are not well versed with Docker. Users who know docker cli well can adapt if needed

Reply:
Good idea! Will try to implement it.